gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcReadChannel.java

```diff
@@ -35,13 +35,17 @@
 import com.google.common.hash.Hashing;
 import com.google.google.storage.v1.GetObjectMediaRequest;
 import com.google.google.storage.v1.GetObjectMediaResponse;
+import com.google.google.storage.v1.GetObjectRequest;
```

Comment:
Can we remove this dependency ? I notice there are no usages, `GetObjectRequest` would be removed the proto going forward as well

Reply:
Sure it can be removed.